src/functions-and-arrays.js

```diff
@@ -1,73 +1,480 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(numOne, numTwo) {
+  if (numOne > numTwo) {
+    return numOne;
+  } else if (numTwo > numOne) {
+    return numTwo;
+  } else {
+    return numOne;
+  }
+}
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(arr) {
+  if (arr.length == 0) {
+    return null;
+  } else if (arr.length == 1) {
```

Comment:
You can actually remove this condition to check if the array only has one element. This will be checked in the for loop so it is redundant code.